airflow/notifiers/email.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow.notifiers import AirflowNotifier
+from airflow.utils.email import send_email
+from airflow.utils.state import State
+
+
+class EmailNotifier(AirflowNotifier):
+
+    def __init__(self, email, notify_on=None):

```

Comment:
`notify_on` is an optional argument. If you don't pass it it will use its defaults. I'm not sure if using class attributes would actually help here.